### PR TITLE
rm linux instructions + update gif

### PR DESCRIPTION
**Pull request by craigmulligan**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -220,6 +214,7 @@
             </tr>
           </tbody>
         </table>
+        <p class="text-center"><a target="_blank" href="https://github.com/resin-io/etcher#debian-and-ubuntu-based-package-repository-gnulinux-x86x64">Looking for debian images?</a><p>
```
Comment (jviotti) [inline index.html:217]: Amazing addition! :)

**Review by jviotti — APPROVED** [review]

File: index.html (hunk 1 above)
Comment (craigmulligan) [reply index.html:217]: haha 🍾

Comment (lurch) [reply index.html:217]: Maybe we should also link to the `brew` instructions for Mac? (and `chocolatey` for Windows too, when that feature gets merged?)

Comment (lurch) [reply index.html:217]: And just to be pedantic, should Debian have a capital `D` ?

**Pull request merged**